graph/db/kv_store.go

```diff
@@ -3157,9 +3176,11 @@ func (c *KVStore) HasLightningNode(_ context.Context,
 
 // nodeTraversal is used to traverse all channels of a node given by its
 // public key and passes channel information into the specified callback.
+//
+// NOTE: the reset param is only meaningful if the tx param is nil.
```

Comment:
nit: expand on this a bit? That if `tx` is _not_ nil, the caller is expected to have provided a reset to the parent `View` or `Update` that is valid/relevant for the whole transaction (which I guess is precisely what you've implemented already anyway).

Reply:
updated 👍